common/src/geometry/creator/include/CGeomtryCreatorScene.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef __NBL_THIS_EXAMPLE_SCENE_H_INCLUDED__
-#define __NBL_THIS_EXAMPLE_SCENE_H_INCLUDED__
+#ifndef _NBL_GEOMETRY_CREATOR_SCENE_H_INCLUDED_
+#define _NBL_GEOMETRY_CREATOR_SCENE_H_INCLUDED_
 
 #include <nabla.h>
```

Comment:
why does this have a `src/geometry/creator/include`  path?

Reply:
resolved